README.md

```diff
@@ -35,12 +35,7 @@ In the development VM, go to:
 
 ```bash
 cd /var/govuk/govuk-puppet/development-vm
-```
-
-Then run:
-
- ```bash
- bowl content-performance-manager
+bundle exec bowl content-performance-manager
```

Comment:
I don't think you should use `bundle exec` with `bowl`, as then things get a bit funny. However, I'm not the best one to ask about this, as I gave up using bowler several years ago...